sos.spec

```diff
@@ -2,21 +2,22 @@ Summary: A set of tools to gather troubleshooting information from a system
 Name: sos
 Version: 4.5.4
 Release: 1%{?dist}
-Group: Applications/System
-Source0: https://github.com/sosreport/sos/archive/%{name}-%{version}.tar.gz
+Source0: https://github.com/sosreport/sos/archive/%{version}.tar.gz
```

Comment:
This is unfortunate but this breaks rpm build from checkout, see e.g. https://github.com/sosreport/sos/pull/3259/checks?check_run_id=13833368826 .

curl-ing https://github.com/sosreport/sos/archive/sos4.5.4.tar.gz really does not work since it is redirected to https://codeload.github.com/sosreport/sos/tar.gz/sos-4.5.4 which does not exist. Contrary to:

    $ curl -vv https://github.com/sosreport/sos/archive/4.5.4.tar.gz 2>&1 | grep location
    < location: https://codeload.github.com/sosreport/sos/tar.gz/refs/tags/4.5.4
    $ curl https://codeload.github.com/sosreport/sos/tar.gz/refs/tags/4.5.4
    .. (this works)
    $

But the stored data is a tarball with content in `sos-4.5.4`. While `Source0` assumes the content to be in `%{version}` dir instead.

We can do some hack in `%py3_install`(?) but.. no idea how, atm.

Reply:
ok, I'll revert that line and keep it "downstream" change only.